From c9d2eb46288cd4f30ed6fec2864b4b78e1e8bc84 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Fri, 8 Mar 2013 10:49:27 +0100
Subject: configure.ac: add missing AM macros
Origin: upstream, https://github.com/legatvs/libquvi/commit/c9d2eb46288cd4f30ed6fec2864b4b78e1e8bc84

This fixes generating configure on systems with recent autotools and
GNU binutils.

Signed-off-by: Martin Herkt <lachs0r@srsfckn.biz>
---
 configure.ac | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure.ac b/configure.ac
index 2fd2d70..f810432 100644
--- a/configure.ac
+++ b/configure.ac
@@ -31,11 +31,14 @@ AC_DEFINE_UNQUOTED([CANONICAL_HOST], "$host", [...])
 AM_INIT_AUTOMAKE([-Wall -Werror foreign dist-bzip2 dist-xz])
 AM_SILENT_RULES([yes])
 
+AM_PROG_AR
+
 LT_INIT()
 LT_PREREQ([2.2])
 
 # Checks for programs.
 AC_PROG_CC
+AM_PROG_CC_C_O
 
 AC_PATH_PROGS([ND], [naturaldocs NaturalDocs], [],
   [$ND_PATH$PATH_SEPARATOR$PATH])
-- 
1.8.4.rc3

